PoseEuler dest
bool set_orientation
---
